app/profile/profile.scss

```diff
@@ -6,5 +12,5 @@
   margin-top: 0px;
 }
 
-@media (max-width: $breakpoint-xl) {
+@media (max-width: 800px) {

```

Comment:
breakpointy też mogą być w stylach zmiennymi, aby utrzymywać ich jednakową wartość i nie ustalać ich z palca, bo przy zmianie podejscia będziesz musiał zmieniać to w N miejscach